docs/builders.rst

```diff
@@ -137,3 +137,24 @@ Format
         }
     }
 
+.. _needumls_builder:
+
+needumls
+--------
+
+The **needumls** builder saves each :ref:`needuml` generated plantuml code to a file, and stores all the files into a single folder during the build.
+The file is created only if the option ``:save:`` from :ref:`needuml` is configured.
+
+The build creates a folder called **needumls** and files that store the generated plantuml code from :ref:`needuml` inside the given build-folder, e.g. `_build/needumls`.
```

Comment:
Rephrase this line and maybe make it simpler? E.g.:
The build creates a folder called **needumls** inside the build folder, for instance _build/needumls.